packages/atlas/src/MainLayout.tsx

```diff
@@ -53,6 +54,20 @@ export const MainLayout: FC = () => {
     onExitClick: () => closeDialog(),
   })
 
+  useEffect(() => {
+    const stopReplay = async () => await SentryLogger.replay?.stop()
+
+    if (location.pathname === absoluteRoutes.viewer.ypp()) {
```

Comment:
why can't we just move this to ypp view?

Reply:
this was the initial Idea, but we have async stopReplay, which would be tricky to execute on component unmount